build/resources/main/templates/error.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8"/>
+  <title>Something went wrong</title>
+  <link rel="stylesheet" href="/style.css"/>
+</head>
+<body>
+  <h1>Oooops, something went wrong.</h1>
```

Comment:
Give a more talkative error message! The fox is cool but use something like thymeleaf to pass the neccesarry error details!